### ISSUE TITLE
[SUGGESTION] Simply Corn, Animania and Salty Mod compat

### ISSUE THREAD

**Issue by Sunconure11** [Snownee/Cuisine#23]
Would it be possible to add support for those three mods?

Furthermore, I have another request, unless it has been done.

Would it be possible to add a system that shuts off redundant crops if another supported mod is using them? So say, there aren't two tomato crops, as an example.

**Comment by Snownee** [comment]
Do you mean Simple Corn? Cuisine adds corn too, if Simple Corn has the same Ore Dict, it should be supported.

**Comment by Sunconure11** [comment]
Yeah, I meant Simple Corn. Also, any plans for serene seasons support?

**Comment by Snownee** [comment]
> Yeah, I meant Simple Corn. Also, any plans for serene seasons support?

I think yes. Need to investigate
